CMakeLists.txt

```diff
@@ -293,14 +293,15 @@ endif (RECOIL_DETAILED_TRACY_ZONING)
 #  So req. checks are done in the build target's CMakeLists.txt.
 find_package(SDL2)
 
-find_package_static(DevIL)
-if    (PREFER_STATIC_LIBS)
-	# dependencies of DevIL
-	find_package_static(JPEG REQUIRED)
-	find_package_static(PNG REQUIRED)
-	find_package_static(TIFF REQUIRED)
-	find_package_static(GIF REQUIRED)
-endif (PREFER_STATIC_LIBS)
+# We don't actually install this in a way that's findable with windows ATM
+if (NOT WIN32)
```

Comment:
If not on windows, then we also depend on pkg-config, so is it really required there?

Reply:
Yes, unfortunately DevIL's cmake find module does not bring in static dependencies; spring won't link without these lines

Reply:
That's a good question -- I think that our problems stem from the fact that we are doing static linking, so the symbols need to be available; with dynamic linking this wouldn't be needed. Actually I think I understand why the condition before was `PREFER_STATIC_LIBS`, so maybe I should revert back to that.

As for PkgConfig, I'm not sure it would actually fix the problem since looking at https://github.com/beyond-all-reason/spring-static-libs/blob/18.04/lib/pkgconfig/IL.pc it does not add to the link line any of the dependent libs

Reply:
I decided to revert back to the old logic on this because I couldn't come up with a convincing reason to change it

Reply:
Ok, annoying :(